### PR TITLE
Create stale bot config

### PR DESCRIPTION
**Pull request by sapk**
https://github.com/probot/stale

For this to work someone of @go-gitea/owners need to activate https://github.com/apps/stale

For starting, I choose 2 month of inactivity and 2 weeks after the issue or PR is closed. I limit to 1 action/hour the bot.

The following labels are skipped :
  - status/blocked 
  - kind/security 
  - lgtm/done
  - reviewed/confirmed
  - priority/critical
  - kind/proposal

### CODE REVIEW

**Comment by sapk** [comment]
List of target : https://github.com/go-gitea/gitea/issues?utf8=%E2%9C%93&q=is%3Aopen+updated%3A%3C2018-09-01+-label%3Astatus%2Fblocked++-label%3Akind%2Fsecurity+-label%3Algtm%2Fdone+-label%3Areviewed%2Fconfirmed+-label%3Apriority%2Fcritical+sort%3Aupdated-asc+-label%3Akind%2Fproposal

**Comment by sapk** [comment]
~We maybe need a label `reviewed/confirmed` to lock some issue that shouldn't be closed.~ Done

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
```
Comment (bkcsoft) [inline .github/stale.yml:8]: I'd set this to a higher value. Like 2 months (at least for PRs...)

Comment (sapk) [reply .github/stale.yml:8]: Done.
I also setup the custom text for PR.

**Comment by lafriks** [comment]
@sapk I have added `reviewed/confirmed` label

**Comment by jonasfranz** [comment]
Also add priority/critical please

**Comment by lafriks** [comment]
@JonasFranzDEV we barely use priority labels that priority/high should be already considired as critical. We could maybe rename it?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  recent activity. It will be closed if no further activity occurs during the next 2 weeks. Thank you
+  for your contributions.
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+# closeComment: >
+#   Your comment here.
```
Comment (kolaente) [inline .github/stale.yml]: Maybe we should add another comment like "This issue was closed because of inactivity. You can repoen it." or so.

Comment (sapk) [reply .github/stale.yml]: Done

**Comment by sapk** [comment]
Should we exclude the label kind/proposal ?

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  daysUntilStale: 60
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs during the next 2 months. Thank you
+    for your contributions.
+  closeComment: >
+    This pull request has been automatically closed because of inactivity.
+    You can re-open it if needed.
+
+# issues:
```
Comment (lafriks) [inline .github/stale.yml]: Remove unneeded commented out lines

**Comment by HoffmannP** [comment]
If these two labels (critical & confirmed) where just added current issues that would be closed in january should be re-checked?

File: .github/stale.yml (hunk 3 above)
Comment (sapk) [reply .github/stale.yml]: Done

**Review by techknowlogick — APPROVED** [review]

**Comment by sapk** [comment]
@HoffmannP The goal is more to help closed issue without responses and auto-close them. This also define a general rule when to consider a issue as stale and can be closed. Actually this is done manually and some can found it hard do find when to close and when not to close.

**Comment by lunny** [comment]
We have 133 open issues with kind/bug label inactive on last 60 days.
https://github.com/go-gitea/gitea/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Akind%2Fbug+updated%3A%3C2018-09-07+

**Comment by lafriks** [comment]
We should probably review these issues and add needed labels for issues that should not be closed before merging this PR.

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  recent activity. It will be closed if no further activity occurs during the next 2 weeks. Thank you
+  for your contributions.
+
+# Comment to post when closing a stale Issue or Pull Request.
+closeComment: >
+  This issue has been automatically closed because of inactivity.
+  You can re-open it if needed.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 1
```
Comment (jonasfranz) [inline .github/stale.yml:40]: Why limit actions per hour to 1 when it defaults to 30?

Comment (sapk) [reply .github/stale.yml:40]: Just in case it goes horribly wrong at first start ^^
We can adjust later.

**Review by jonasfranz — APPROVED** [review]

**Pull request merged**